### PR TITLE
Add beamer option to show notes on second screen

### PR DESCRIPTION
**Pull request by bersace**
Hi,

To use speaker mode in pdfpc or pympress, i need pandoc to render notes on *second screen*. I suggest to add a beamer specific option for this, since this behaviour is completely specific to beamer.

What do you think of this ?

Thanks for pandoc !

Regards,

### CODE REVIEW

**Comment by bersace** [comment]
I'm wondering how to document that. Can you give me a clue ? Thanks !

**Comment by bersace** [comment]
Ok, i got it. It's documented in MANUAL :-)

**Comment by mb21** [comment]
Please use pandoc-discuss for questions like this...

**Comment by jgm** [comment]
@mb21 It looks like this is a PR, with some code, not a question.  The PR should be considered.

File: MANUAL.txt [hunk 1]
```diff
@@ -1542,6 +1542,10 @@ LaTeX variables are used when [creating a PDF].
 `natbiboptions`
 :   list of options for natbib.
 
+`notes2screen`
```
Comment (jgm) [inline MANUAL.txt]: Would it make more sense to call this just `notes`?
Or is there also some way to display notes on the same screen in beamer?

Comment (bersace) [reply MANUAL.txt]: @jgm I didn't find a best name. notes can be on separate slides.

Maybe we could just add a raw option `shownotes` which is actually just a passthru for `\setbeameroption` ?

Do you have a better idea ? I'm ok with `notes` if you confirm this choice.

Comment (jgm) [reply MANUAL.txt]: Maybe for maximum generality, something like

```
$if(beameroptions)$
$for(beameroption)$$beameroption$$sep$,$endfor$
$endif$
```

then you could have in your YAML
```
beameroptions:
- show notes on second screen=top
```

Comment (bersace) [reply MANUAL.txt]: @jgm that's perfect ! Just like classoptions. I'll do this.

**Comment by bersace** [comment]
Closing in favor fo #4359 . Thanks @jgm for the review.

**Pull request closed without merging**